tests/tests.json

```diff
@@ -28,6 +28,19 @@
                 "expect": 0
             },
             {
+                "test name": "Subcommands with name and alias map correctly",
+                "file name": "testSubCommands",
+                "args": "a -f foo bar baz --testing",
+                "expect": 0
+            },
+            {
+                "test name": "Subcommands with an array of aliases map correctly",
+                "file name": "testSubCommands",
+                "args": "p",
```

Comment:
Here is the second part of what to fix to make our tests pass: `"args": "p --testing"` .